versions.props

```diff
@@ -6,7 +6,7 @@ com.google.errorprone:error_prone_core = 2.0.18
 com.google.guava:guava = 20.0
 com.jcraft:jzlib = 1.1.3
 com.netflix.feign:feign-*= 8.17.0
-com.palantir.remoting-api:* = 1.7.0
+com.palantir.remoting-api:* = 1.8.0-15-gb1c30f7
```

Comment:
I've just chopped a release: https://github.com/palantir/http-remoting-api/releases/tag/1.9.0

Wanna do the updateLocks saveLocks thing again to try to bring back that `okhttp-clients/versions.lock` file?